projects/packages/my-jetpack/src/products/class-stats.php

```diff
@@ -43,6 +44,20 @@ class Stats extends Module_Product {
 	 */
 	public static $plugin_filename = self::JETPACK_PLUGIN_FILENAME;
 
+	/**
+	 * Stats only requires site connection, not user connection
+	 *
+	 * @var boolean
+	 */
+	public static $requires_user_connection = true;
```

Comment:
Wasn't this set to `false` in the first revision? Curious why it is `true` now?

Reply:
I answered this in https://github.com/Automattic/jetpack/pull/35355#discussion_r1479115067 I left above.